runtime/src/constants.rs

```diff
@@ -277,6 +278,13 @@ pub mod staking {
 			.max
 			.get(DispatchClass::Normal);
 		pub const OffchainRepeat: BlockNumber = 5;
+
+		// Note: the EPM in this runtime runs the election on-chain. The election bounds must be
```

Comment:
Does this mean anything for us?

Reply:
I think it applies for us even if we won't reach the limits so i took the comment with the line